2d/caissonBreakwater/sliding/tank.py

```diff
@@ -0,0 +1,775 @@
+from proteus import Domain, Context
```

Comment:
Before pushing it, did you change anything? It looks like the latest version I used by the way. Good.

Reply:
I just changed about "setSprings" and "setabsorptiozones" in tank.py because as you can remember there was a little error. No more than this.